ujin/programmers/day05/이상한-문자-만들기.js

```diff
@@ -0,0 +1,16 @@
+function solution(s) {
+  return s
+    .split(" ")
+    .map(word => {
+      let result = "";
+      for (let i = 0; i < word.length; i++) {
+        if (i % 2 != 0) {
```

Comment:
1. 0 이상의 정수 (i) 를 2로 나눈 나머지 값은 항상 0 이거나 1 입니다. 0 과 1 은 `false` 와 `true` 의 의미를 갖기 때문에 `!= 0` 로 `0과 같이 않다` 라는 비교연산을 하지 않아도 됩니다. 
2. JavaScript 에서는 `==` 대신 `===` 연산자, `!=` 대신 `!==` 연산자를 사용하는 것이 좋습니다.
```suggestion
        if (i % 2) {
```